native/themes/markdown-styles.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import { markdownStyle } from '../themes/colors';
+
+const unboundStyles = {
+  autolink: {
+    color: 'link',
+    textDecorationLine: 'underline',
+  },
+  noMargin: {
+    marginBottom: 0,
+    marginTop: 0,
+  },
+  paragraph: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+  },
+  paragraphCenter: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    textAlign: 'center',
+  },
+  text: {
+    color: 'listForegroundLabel',
+    fontFamily: 'Arial',
+    fontSize: 18,
+  },
+  textRow: {
+    flexDirection: 'row',
+  },
+  view: {
+    alignSelf: 'stretch',
+  },
+};
+
+export type Style = $Exact<typeof unboundStyles>;
```

Comment:
1. Let's name this export something more specific. How about `MarkdownStyles`
2. Is `typeof unboundStyles` not exact already? You can figure this out by running:

`flow type-at-pos themes/markdown-styles.js 5 7`